docs/install.rst

```diff
@@ -102,6 +102,10 @@ your site::
 
     pelican-quickstart
 
+If it is running inside a virtual environment it will try to detect the path
+associated to it and save the project there. A custom path can be configured
+using the ``--path`` flag.
+
```

Comment:
In the interest of extra clarity, what do you think about this proposed alternate text?
> If run inside an activated virtual environment, `pelican-quickstart` will look for an associated project path inside `$VIRTUAL_ENV/.project`. If that file exists and contains a valid directory path, the new Pelican project will be saved at that location. Otherwise, the default is the current working directory. To set the new project path on initial invocation, use: `pelican-quickstart --path /your/desired/directory`

Reply:
Yeah, that sounds good!
I've updated the text in 66818be4418c813ab939efc02f63d7026ef80998, do you want me to squash the commits?

Reply:
Thanks! No need — I'll take it from here.